src/CMakeLists.txt

```diff
@@ -486,8 +486,6 @@ if(FLB_SHARED_LIB)
   if (MSVC)
     set_target_properties(fluent-bit-shared
       PROPERTIES PDB_NAME fluent-bit.dll)
-    target_link_options(fluent-bit-shared
-      PUBLIC /pdb:$<TARGET_PDB_FILE:fluent-bit-shared>)
```

Comment:
It seems mostly good but I have a question for this change.
We publish pdbs which contain debug information for digging issues.
So, we heavily depending on the generated pdbs.
If the lines are removed, the pdbs are still going to be generated?
If no, could you restore the change?

Reply:
Yes, this is covered by the `/DEBUG` linker option in `CMAKE_EXE_LINKER_FLAGS` (in the root `CMakeLists.txt`, not altered) and the modified `CMAKE_SHARED_LINKER_FLAGS`. However, the pdb for the dll is not distributed, only the one for the executable (no change from before). This is not an issue when debugging `fluent-bit.exe` as it does not use the dll anyway since it is linked with the static library but is obviously not great for anyone wanting to debug the dll.

As an aside, supplying the dll's importlib and and pdb would be useful for anyone wanting to use the distributed dll.

Reply:
Got it. Let me check for pdbs with generated packages for Windows.